### ISSUE TITLE
Cannot find module '!!../css-loader/index.js!./croppie.css'

### ISSUE THREAD

**Issue by sombriks** [jofftiquez/vue-croppie#27]
Hello,

i have a browserify setup using require/exports setup.

when i try to do a **Vue.use(VueCroppie)**, i get the following error:

```bash
Cannot find module '!!../css-loader/index.js!./croppie.css'
vue-img-upload/node_modules/vue-croppie/dist
```

here is what i am doing: 

```javascript
const VueCroppie = require("vue-croppie")
Vue.use(VueCroppie)
```

Any guidance is welcome.

**Comment by jofftiquez** [comment]
Hello @sombriks  I'm not entirely sure what's wrong but it might be related to the `css-loader`. Sorry for the late response. Please let me know of the progress. Thanks.

**Comment by sombriks** [comment]
Hi @jofftiquez !

I am pretty sure this is related to the browserify.

I've made a sample project if you want to see the error: https://github.com/sombriks/sample-vue-croppie-browserify

Since there is no css loader on browserify projects (the browserify transform works in another way) i think there's not much to do.

I've used webpack-ish projects in the past (for example vue itlself works fine with browserify and muse-ui too), yet i can't tell what they have to allow browserify to correctly consume them.
